routes/v1.js

```diff
@@ -502,52 +521,60 @@ api.declare({
     // otherwise the task would have been created with defineTask, and we don't
     // offer an idempotent operation in that case
     if (!_.isEqual(taskDef, def) || task.runs.length === 0) {
-      return res.reportError(
-        "RequestConflict", [
-          "taskId {{taskId}} already used by another task.",
-          "This could be the result of faulty idempotency!"
-        ].join('\n'),
-        {
-          taskId,
-        });
+      return res.reportError('RequestConflict', [
+        "taskId {{taskId}} already used by another task.",
+        "This could be the result of faulty idempotency!",
+        "Existing task definition was: {{existingTask}}",
```

Comment:
This is going to be inserting a big json blob here, right? Maybe we can stick it on it's own line or think about moving in the direction of using Markdown here that we've talked before, right? We could stick this in a code block.